alliance/config/settings.py

```diff
@@ -217,13 +254,28 @@ def template_folder_paths(app):
 # Miscellaneous configuration
 ################################################################################
 
-ROOT_URLCONF = 'alliance.config.urls'
+ROOT_URLCONF = 'config.urls'
 LANGUAGE_CODE = 'en-us'
 TIME_ZONE = 'America/Chicago'
 USE_I18N = True
 USE_L10N = True
 USE_TZ = True
-WSGI_APPLICATION = 'alliance.wsgi.application'
 SESSION_COOKIE_AGE = 10 * 60  # 10 minutes
 SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
-ALLOWED_HOSTS = ['*']
\ No newline at end of file
+ALLOWED_HOSTS = ['*']
+
+
+################################################################################
+# Custom configuration
+################################################################################
+
+# Look to see if there is a `local.py` file in the `config` folder, if so, load
+# it up and override all the things. We do NOT use these settings if the
+# `ALLIANCE_CUSTOM_SETTINGS` are set to False.
+has_local_settings = os.path.exists(os.path.join(CONFIG_ROOT,
+                                                 'local_settings.py'))
+use_custom_settings = os.getenv('ALLIANCE_CUSTOM_SETTINGS', 'True')
+
+if has_local_settings and use_custom_settings == 'True':
```

Comment:
@kdflint This can be set to something else to prevent overriding the `settings.py` file.